src/Api/Controllers/ConfigController.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.Reflection;
+using Bit.Api.Models.Response;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Bit.Api.Controllers
+{
+    [Route("config")]
+    public class ConfigController : Controller
+    {
+        public const string GIT_HASH_ASSEMBLY_KEY = "GitHash";
+
+        public ConfigController()
+        {
+
+        }
+
+        [HttpGet("")]
+        public ConfigResponseModel GetConfigs()
+        {
+            ConfigResponseModel response = new ConfigResponseModel();
+            var customAttributes = Assembly.GetEntryAssembly().GetCustomAttributes<AssemblyMetadataAttribute>();
+
+            if (customAttributes.Count() > 0)
+            {
+                response.GitHash = customAttributes.Where(i => i.Key == GIT_HASH_ASSEMBLY_KEY).First().Value;
+            }
```

Comment:
I like this idea, only question is what do we do with the other references to the assembly? I think there are roughly ~18 in various places, so it will add some time updating all of that. Can it be a slow rollout towards the Assembly Helper class or do we need to do it all at once?

Reply:
I think we can create a helper class that contains just the `GetVersion` and `GetGitHash` (or whatever we call it) to start with.

Reply:
Great idea. Updated.